Source/__tests__/Application.test.js

```diff
@@ -1,46 +1,44 @@
-import { assertEqual } from './assert'
+/* globals expect */
```

Comment:
yeah it's a jest convention. The regex is `(/__tests__/.*|(\\.|/)(test|spec))\\.jsx?$`. So you can also just call your file `*.test.js` and it will get picked up. I like to do both :)